include/yateto/TensorView.h

```diff
@@ -326,6 +326,20 @@ namespace yateto {
       return m_values[addr];
     }
 
+    bool isInRange(uint_t row, uint_t col) {
+      assert(col >= 0 && col < this->shape(1));
+      uint_t addr = m_colPtr[ col ];
+      uint_t stop = m_colPtr[ col+1 ];
+      while (addr < stop) {
```

Comment:
Why no `for` loop here?

Reply:
Because it's literally the same loop as in the function directly above it?

Reply:
Ok, works for me